en/docs/assets/attachments/InstallAndSetup/apim200_to_apim260_gateway_artifact_migrator.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Shall we rename to "apim200_to_apim300_gateway_artifact_migrator.sh"?

Reply:
Fixed